src/components/ResultContainer/ResultContainer.tsx

```diff
@@ -26,13 +26,36 @@ export default function ResultContainer(props: IResultContainerProps) {
     renderResultCardPriceDetails,
   } = resultCardOptions || {};
   const numberOfResults = state?.quiz.results?.response?.results?.length;
+  const resultsConfig = state?.quiz.resultsConfig?.desktop;
```

Comment:
Forgot why we agreed on this structure. But this value wil be used for all viewports.

Reply:
Why is it named `desktop` if it's being used for all viewports? I think it will be okay for UI library users since they don't interact with it directly but for anyone doing an API integration, this will be confusing

Reply:
I believe the idea behind was that users in the future can have different texts for different viewports. For example a longer description for bigger screens and shorter one for mobile and tablets. But for an mvp, only the desktop implementation is available as of now.